cmd/thanos/tools_bucket.go

```diff
@@ -84,7 +84,7 @@ func registerBucketVerify(m map[string]setupFunc, root *kingpin.CmdClause, name
 		Short('r').Default("false").Bool()
 	issuesToVerify := cmd.Flag("issues", fmt.Sprintf("Issues to verify (and optionally repair). Possible values: %v", allIssues())).
 		Short('i').Default(verifier.IndexIssueID, verifier.OverlappedBlocksIssueID).Strings()
-	idWhitelist := cmd.Flag("id-whitelist", "Block IDs to verify (and optionally repair) only. "+
+	idAllowlist := cmd.Flag("id-allowlist", "Block IDs to verify (and optionally repair) only. "+
```

Comment:
Is there any additional thing even necessary? Just `id` or `ids` would be even better no?

Reply:
lol, kind of true (: